src/components/folio-number/FolioNumber.vue

```diff
@@ -1,68 +1,98 @@
 <template>
   <div id="folio-number">
     <!-- Display Folio Number -->
-    <v-row v-if="!isEditing" id="display-folio-number" no-gutters>
+    <v-row
+      v-if="!isEditing"
+      id="display-folio-number"
+      no-gutters
+    >
       <v-col cols="3">
         <label>
           <strong>Business Folio or<br>Reference Number</strong>
-          <v-flex>
-            <v-chip v-if="hasFolioNumberChanged" x-small label color="primary" text-color="white">
-              {{editedLabel}}
+          <v-col>
```

Comment:
ESlint made this change (not me). I need to test it.

Reply:
`v-flex` is deprecated. All that was needed here is a div so I updated this (in next commit).